### DISCUSSION TITLE
1.39.1

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#23474] [General]
Feel free to discuss the [v1.39.1](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.39.1) release here!

**Comment by eerison** [comment]
Hey @Koenkk 

I just updated to the last version 1.39.1, and the docker container does not start, I back to version 1.39.0 and it is working.

I don't see any log message, just `using /app/data as directory` and keep restarting

**Reply by eerison** [reply]
I thought it could be a device that was changed in this release, then I removed the device (sprinkler) and updated to 1.39.1, But I got the same issue :/

**Reply by MHerbst** [reply]
I have the same problem with the Home Assistant Add-On. In my case, socat prevented the start. After changing the port configured in the socat slave setting zigbee2mqtt started.

**Reply by RaoulV82** [reply]
Looks like a super update... I'm sure many people will appreciate the easier way to channel change, but unfortunately I've had the same issue as described above and rolled back 
Do I simply need to reinstall and put whatever 4 digit slave port I have now into the config?

**Reply by eerison** [reply]
I'm using slz06 via serial port: `tcp://myip:1234`

I saw in the docs that it is recommended mdns
https://www.zigbee2mqtt.io/guide/configuration/adapter-settings.html#mdns-zeroconf-discovery, I will give a try, if it works like this

**Reply by eerison** [reply]
I tried this auto discovery and I got the same issue, I back to use tcp//ip:port and version 1.39.0

**Reply by Koenkk** [reply]
Please open a [new issue](https://github.com/Koenkk/zigbee2mqtt/issues) with the debug logging attached.

See [this](https://www.zigbee2mqtt.io/guide/configuration/logging.html) on how to enable debug logging.

**Reply by eerison** [reply]
Opened the issue here
https://github.com/Koenkk/zigbee2mqtt/issues/23635

**Comment by hkchew03** [comment]
This version mess up illuminance values of presence sensor model ZG-205Z/A. Values drop from thousand to double digits. Restore back to 1.39.0.1 fixed it.

**Reply by Koenkk** [reply]
That's expected, see https://github.com/Koenkk/zigbee2mqtt/issues/19803

**Comment by RubenKelevra** [comment]
@Koenkk Time to unpin and pin 1.40.0 instead :)
